examples/custom_user_data.rs

```diff
@@ -0,0 +1,22 @@
+use std::path::PathBuf;
+use wry::{Application, Attributes, Result};
+
+fn main() -> Result<()> {
+  let mut app = Application::new()?;
+
+  let test_path = PathBuf::from(env!("OUT_DIR"));
```

Comment:
We should provide an example path imho. An environment variable is less clear.

Reply:
Something like `./target/webview_data` is good for you?

Reply:
sure!

Reply:
Perfect! Just tested and it works fine. 
Had to have an absolute path. 

So maybe it'll be important to tell it in the doc.